utils/src/ooinstall/openshift_ansible.py

```diff
@@ -130,10 +130,10 @@ def run_main_playbook(hosts, hosts_to_run_on, verbose=False):
     inventory_file = generate_inventory(hosts_to_run_on)
     if len(hosts_to_run_on) != len(hosts):
         main_playbook_path = os.path.join(CFG.ansible_playbook_directory,
-                                          'playbooks/common/openshift-cluster/scaleup.yml')
+                                          'playbooks/byo/openshift-cluster/scaleup.yml')
     else:
         main_playbook_path = os.path.join(CFG.ansible_playbook_directory,
-                                          'playbooks/byo/config.yml')
+                                          'playbooks/byo/openshift-cluster/config.yml')
```

Comment:
@brenton yes, the playbooks/byo/config.yml plyabook simply includes playbooks/byo/openshift-cluster/config.yml and was basically only left in place because the training/docs was pointing to it already.